.pre-commit-config.yaml

```diff
@@ -1,6 +1,10 @@
+ci:
+    autofix_commit_msg: 'refactor(lint): apply automatic lint fixes'
+    autoupdate_commit_msg: 'chore(deps): bump pre-commit linter versions'
```

Comment:
BTW out of curiosity, could you link me the documentation for how to use the `ci` block? I haven't seen this before and am having trouble finding it in the `pre-commit` documentation.

Reply:
It's part of the free CI offering for `pre-commit`, [details here](https://pre-commit.ci/). It's only available for OSS, so we haven't been using it for our other internal projects, but I figured enabling it here could be nice. Besides running the linters, which we already do in Circle, it'll also keep linter versions up-to-date (which Renovate doesn't support yet) and automatically push commits fixing lint errors, where possible, to open PRs.